src/AppRoutes.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Routes, Route, Navigate, BrowserRouter } from "react-router-dom";
+import LandingPage from "./pages/LandingPage";
+import HomePage from "./pages/HomePage";
+import LoginPage from "./pages/LoginPage";
+import { useContext } from "react";
+import { UserContext } from "./contexts/UserContext";
+
+function AppRoutes() {
+  const { userInfo } = useContext(UserContext);
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          element={userInfo == null ? <LandingPage /> : <Navigate to="/homepage" />}
```

Comment:
You're using userInfo == null in some places and userInfo in others. It’s better to use a consistent approach throughout, either checking explicitly for null or using a truthy/falsy check.

Reply:
Thanks for the suggestions. I have made the change